src/helpers/connectors.ts

```diff
@@ -0,0 +1,78 @@
+import { getUrl } from '@/helpers/utils';
+import metamaskIcon from '@/assets/connectors/metamask.png';
+import walletconnectIcon from '@/assets/connectors/walletconnect.png';
+import coinbaseIcon from '@/assets/connectors/coinbase.png';
+import portisIcon from '@/assets/connectors/portis.png';
+import gnosisIcon from '@/assets/connectors/gnosis.png';
+import argentxIcon from '@/assets/connectors/argentx.png';
+
+export default {
+  injected: {
+    id: 'injected',
+    name: 'MetaMask',
+    type: 'injected',
+    root: 'ethereum',
+    icon: metamaskIcon
+  },
+  walletconnect: {
+    id: 'walletconnect',
+    name: 'WalletConnect',
+    network: '1',
+    icon: walletconnectIcon,
+    options: {
+      rpc: {
+        '1': 'https://cloudflare-eth.com',
+        '4': 'https://eth-rinkeby.alchemyapi.io/v2/twReQE9Px03E-E_N_Fbb3OVF7YgHxoGq',
+        '42': 'https://eth-kovan.alchemyapi.io/v2/QCsM2iU0bQ49eGDmZ7-Y--Wpu0lVWXSO'
```

Comment:
These networks are shutdown we can remove them
```suggestion
        '1': 'https://cloudflare-eth.com'
```